pl_examples/test_examples.py

```diff
@@ -25,6 +28,35 @@ def test_gpu_template(cli_args):
         run_cli()
 
 
+@pytest.mark.parametrize('cli_args', [
+    '--max_epochs 1 --gpus 1',
+    '--max_epochs 1 --gpus 1 --evaluate',
+])
+@pytest.mark.skipif(not torch.cuda.is_available(), reason="test requires GPU machine")
+def test_imagenet(tmpdir, cli_args):
+    """Test running CLI for the ImageNet example with default params."""
+
+    from pl_examples.domain_templates.imagenet import run_cli
+
+    # https://github.com/pytorch/vision/blob/master/test/fakedata_generation.py#L105
+    def _make_image(file_path):
+        Image.fromarray(np.zeros((32, 32, 3), dtype=np.uint8)).save(file_path)
+
+    for split in ['train', 'val']:
+        for class_id in ['a', 'b']:
+            os.makedirs(os.path.join(tmpdir, split, class_id))
+            # Generate 5 black images
+            for image_id in range(5):
+                _make_image(os.path.join(tmpdir, split, class_id, str(image_id) + '.JPEG'))
+
+    cli_args = cli_args.split(' ') if cli_args else []
+    cli_args += ['--data-path', str(tmpdir)]
+    cli_args += ['--default_root_dir', str(tmpdir)]
+
+    with mock.patch("argparse._sys.argv", ["any.py"] + cli_args):
+        run_cli()
+
+
 # @pytest.mark.parametrize('cli_args', ['--max_epochs 1 --max_steps 3 --num_nodes 1 --gpus 2'])
```

Comment:
Can we remove these comments if we don't need them?

Reply:
Yeah I was wondering myself. They are outdated and have to be rewritten anyway if we ever get multi-node ddp testing.